### ISSUE TITLE
adb.py install_sample don't work

### ISSUE THREAD

**Issue by migel571** [idanr1986/cuckoo-droid#71]
It prints out the message "installing sample on emulator: pm install" + path "and stops at this. Wait for the analysis to end and create a report, but does not install the application on the device." What can be the reason for this error? Can use "proc = subprocess.Popen" (["/ system / bin / pm", "install", path], stdout = subprocess.PIPE, stderr = subprocess.PIPE) ". And another question about "Dropped Files" - they never exist!

**Comment by krishnadev6** [comment]
@migel571 Is adb installed in your system?

**Comment by migel571** [comment]
yes! sometimes it is installed application on the device. sometimes it is not installed. and I did not find any regularities.

**Comment by migel571** [comment]
@krishnadev6  yes! sometimes it is installed application on the device. sometimes it is not installed. and I did not find any regularities.

**Comment by wickywaka** [comment]
That's the problem with os.popen. It did same for me, sometimes it works and sometime it doesn't. You have to replace it with subprocess.popen. Have a look at #53

**Comment by wickywaka** [comment]
Have a look here, you have to make same kind of change as shown in this video for install too:
https://www.youtube.com/watch?v=HRBIZaA5N-w
